docs/en/install-upgrade/upgrading-elasticsearch.asciidoc

```diff
@@ -109,6 +109,8 @@ jobs with large model states.
 . [[upgrade-node]] *Shut down a single node*.
 +
 --
+To shut down a single node depends on what is currently used to run {es}. For example, if using `systemd` or SysV `init` see the commands below.
```

Comment:
```suggestion
To shut down a single node depends on what is currently used to run {es}. For example, if using `systemd` or SysV `init` run the commands below.
```
Our accessibility guidelines (which I can't find at the moment or I would share the link) recommend not to use `see`. Instead we typically use `check`, `refer to`, or other.

Reply:
Ahhhh ok got it. I'll see if I can hunt them down. Thank you for catching that!